src/core/LoadBalancer.ts

```diff
@@ -68,17 +71,38 @@ export default class LoadBalancer {
         await this.page.keyboard.press("Enter");
 
         // Make sure that the user was correctly assigned
-        const gradersAssigned = await this.page.$$eval(
-            "ul .search-choice span",
-            (el) => el.map((grader) => grader.textContent),
-        );
+        const gradersAssigned = await this.getAssignedGraders();
         if (!gradersAssigned.includes(grader.name)) {
             throw new UserError(
                 `Couldn't assign ${grader.name}. Please verify there's a Greenhouse user with this name`,
             );
         }
     }
 
+    /**
+     * Get currently assigned graders
+     */
+    private async getAssignedGraders() {
+        return await this.page.$$eval("ul .search-choice span", (el) =>
+            el.map((grader) => grader.textContent),
+        );
+    }
+
+    /**
+     * Returns true if assigned graders has at least graders count
+     * of already assigned graders.
+     * 
+     * Exma
+     */
+    private async gradersAlreadyAssigned() {
+        const gradersAssigned = await this.getAssignedGraders();
+        const seenGraders = this.graders.filter((g) => 
+                gradersAssigned.includes(g.name)
+            ).length
+
+        return seenGraders > 0 && seenGraders === this.gradersCount;
```

Comment:
All written interviews have at least one grader as it is usually set to the hiring lead by default.